pre-docs/bolt_installing.md

```diff
@@ -78,17 +78,17 @@ puppet5-release-el-7.noarch.rpm.
 - Enterprise Linux 6
   ```
   sudo rpm -Uvh  https://yum.puppet.com/puppet5/puppet5-release-el-6.noarch.rpm
-  sudo yum install bolt
+  sudo yum install puppet-bolt
   ```
 - Enterprise Linux 7
   ```
   sudo rpm -Uvh https://yum.puppet.com/puppet5/puppet5-release-el-7.noarch.rpm
-  sudo yum install bolt
+  sudo yum install puppet-bolt
   ```
 - SUSE Linux Enterprise Server 12
   ```
   sudo rpm -Uvh https://yum.puppet.com/puppet5/puppet5-release-sles-12.noarch.rpm
-  sudo zypper install bolt
+  sudo zypper install puppet-bolt
```

Comment:
Do we want to change the el package name as well for consistency? I assumed we would but suppose we don't strictly need to.

Reply:
Yes, I think consistency is more useful. And because the other bolt package is being added to Fedora, it'll probably make it into CentOS and RedHat eventually.

Consistency is the main argument for macOS and Windows.

Reply:
I'm losing my mind -- I'm not sure why but I thought we didn't change it for el and was wondering why.